fusedrug/data/interaction/drug_target/datasets/dti_binding_dataset.py

```diff
@@ -493,9 +498,13 @@ def _load_dataframes(
         )
 
     if combine:
-        ligand_id_key = "ligand_id"
+        ligand_id_key = _ligands.index.name
```

Comment:
Are we sure that ligand_id is the same as index in _ligands (same goes for targets)? Do we enforce it anywhere?

Reply:
In this code it was set to be (see few lines before)
and you used a hardcoded name.
the reason I do this is because while it was set to be the index it was done with drop=True, so it is no longer a column, which I wanted to assume in my check